### ISSUE TITLE
[Enhancement] Add biomedical "zing" to graphics used on Home page

### ISSUE THREAD

**Issue by vpchung** [Sage-Bionetworks/sage-monorepo#1836]
## Sage Design feedback

Current home page graphic:

![Image](https://github.com/Sage-Bionetworks/sage-monorepo/assets/9377970/6a90864e-17d7-483e-8270-1bde01169206)

At a quick first glance, it is not clear that this is a website dedicated to _biomedical_ challenges, at least, not until some text is read.  

Consider adding a DNA helix or some other type of biomedical-related graphics/image to the main graphic.

**Comment by tschaffter** [comment]
A hint is given by the dynamic search term examples we provide but this is very subtle. We also have another ticket about modifying the content of the paragraph, which gives us an opportunity to be more explicit. But I agree that it would be nice to have our focus on biomedical challenges reflected on the illustration.

**Comment by rrchai** [comment]
> Consider adding a DNA helix or some other type of biomedical-related graphics/image to the main graphic.

It's a good idea. In addition, the text "biomedical" can be visually emphasis in the paragraph as well.

**Comment by tschaffter** [comment]
Blocked until we have the new plot on the Home page. Do we show the plot as the first/main illustration?

**Comment by tschaffter** [comment]
## Update 2023-08-28

This ticket is blocked until we have the "challenges per year" plot and then decide whether we still need the illustration that this ticket proposes to replace.

**Comment by tschaffter** [comment]
Moving to Sprint 23.09

**Comment by vpchung** [comment]
Verena to do a quick mockup of new home page design, replacing main illustration with metrics + plot.

**Comment by vpchung** [comment]
Following our [recent decision of the home layout](https://github.com/Sage-Bionetworks/sage-monorepo/pull/2136#issuecomment-1731781362), here is my proposed illustration for the landing section: 


![Screenshot 2023-09-25 at 10 47 21 AM](https://github.com/Sage-Bionetworks/sage-monorepo/assets/9377970/ae9897ef-21ed-442d-b588-97dcf573c993)

**Closeup**
![Screenshot 2023-09-25 at 10 45 06 AM](https://github.com/Sage-Bionetworks/sage-monorepo/assets/9377970/9050e8b1-027d-49ea-a61c-b5d6540b0d06)

**Comment by tschaffter** [comment]
Thanks for making this draft! Below are some suggestions:

- My first thought was that the image was too dense.
- I would use the magnifying glass to highlight elements, either by placing elements inside and/or around the magnifying glass. In the current draft, it may look like the character is looking at nothing.

**Comment by vpchung** [comment]
Thanks for the feedback, @tschaffter .  Here are some more iterations:

## Option A
Illustrations used: heart, xray scans, germs
Illustrations located inside magnifying glass

![Screenshot 2023-09-25 at 11 27 17 AM](https://github.com/Sage-Bionetworks/sage-monorepo/assets/9377970/0bdfca9c-0399-4ccb-ba78-db718c2575cc)
![Screenshot 2023-09-25 at 11 27 23 AM](https://github.com/Sage-Bionetworks/sage-monorepo/assets/9377970/00a2be7d-4c9b-4f43-b13d-5a4075e561d6)

## Option B
Illustrations used: heart, pill bottle, germs
Illustrations located inside magnifying glass

![Screenshot 2023-09-25 at 11 28 43 AM](https://github.com/Sage-Bionetworks/sage-monorepo/assets/9377970/76f10698-4978-4150-9dd2-5b6b1d9df726)
![Screenshot 2023-09-25 at 11 28 54 AM](https://github.com/Sage-Bionetworks/sage-monorepo/assets/9377970/280ce63f-3cf9-4814-8118-167db5c48e83)

## Option C
Illustrations used: heart, pill bottle, germs
Illustrations located outside magnifying glass

![Screenshot 2023-09-25 at 11 30 38 AM](https://github.com/Sage-Bionetworks/sage-monorepo/assets/9377970/671ad442-7221-45a5-ab46-767bfedf64fb)
![Screenshot 2023-09-25 at 11 30 45 AM](https://github.com/Sage-Bionetworks/sage-monorepo/assets/9377970/7eabd1f0-2c8a-482b-ba69-8e0002c067cd)
